github-pr-reviewer/code_reviewer/github_code.py

```diff
@@ -0,0 +1,28 @@
+from github import Github
+
+from code_reviewer.reviewer import CodeReviewer
+
+class GithubAPI:
+    def __init__(self, access_token):
+        self.g = Github(access_token)
+        self.code_reviewer = CodeReviewer()
+    
+    def write_comments_for_pr(self, repo_name, pr_number):
+        repo = self.g.get_repo(repo_name)
+        pr = repo.get_pull(pr_number)
+        commit = repo.get_commit(pr.head.sha)
+        files = pr.get_files()
+        for file in files:
+            if file.status == 'added' or file.status == 'modified':
+                comments = self.code_reviewer(file.filename, file.patch)
+                for filename, position, comment in comments:
+                    pr.create_review_comment(body=comment, commit_id=commit, path=filename, position=position)
+        return 200
+
```

Comment:
Instead of returning an integer (200) as a success indicator, consider returning a more descriptive message using a string or a custom response object.